BWT/Program.cs

```diff
@@ -0,0 +1,133 @@
+﻿/// <summary>
+/// BWT direct conversion function/
+/// </summary>
+/// <param name="inputString">Converted string.</param>
+/// <returns>The converted string and the position of the end of the string.</returns>
+(string, int) BWT(string inputString)
+{
+    int[] suffixArray = new int[inputString.Length];
```

Comment:
```suggestion
   var suffixArray = new int[inputString.Length];
```
Не пишите тип дважды